pkg/core/interop/context.go

```diff
@@ -65,6 +65,8 @@ type Function struct {
 	ID   uint32
 	Name string
 	Func func(*Context, *vm.VM) error
+	// DisallowCallback is true iff syscall can't be used in a callback.
+	DisallowCallback bool
```

Comment:
For `Neo.Native.Deploy` and `Neo.Native.Call` callbacks should also be disallowed.

Reply:
FIxed.